.github/workflows/image.yaml

```diff
@@ -84,18 +93,6 @@ jobs:
     steps:
       - uses: actions/checkout@v4
         name: Check out code
```

Comment:
What version is checked out here? Should we align this with the short sha?

Reply:
would make sense in a follow-up PR. The defaults are ok, but moving forward when writing more reusable workflows, it will make more sense 

https://github.com/actions/checkout?tab=readme-ov-file#usage

Reply:
Shure. Could you please clarify which verison is checked out by default.

Reply:
```yaml
    # The branch, tag or SHA to checkout. When checking out the repository that
    # triggered a workflow, this defaults to the reference or SHA for that event.
    # Otherwise, uses the default branch.
```

Reply:
is the SHA of the commit that triggered the action, unregarding of the branch (that's why copy-pr-bot works)